engine-tests/src/test/java/org/terasology/world/chunks/localChunkProvider/LocalChunkProviderTest.java

```diff
@@ -128,8 +128,8 @@ void testGenerateSingleChunk() throws InterruptedException, ExecutionException,
                     Event mustBeOnLoadedEvent = eventArgumentCaptor.getAllValues().get(1);
                     Assertions.assertTrue(mustBeOnLoadedEvent instanceof OnChunkLoaded,
                             "Second world event must be OnChunkLoaded");
-                    Assertions.assertEquals(((OnChunkLoaded) mustBeOnLoadedEvent).getChunkPos(),
-                            chunkPosition,
+                    Assertions.assertEquals(JomlUtil.from(chunkPosition),
+                            ((OnChunkLoaded) mustBeOnLoadedEvent).getChunkPos(),
                             "Chunk position at event not expected");
```

Comment:
The order for assertions is _expected_, _actual_, _message_, so I corrected this while adding `JomlUtil` conversion to compare vectors of the same type.